pkg/controller/plan/adapter/converter.go

```diff
@@ -134,7 +137,7 @@ func createConvertJob(pvc *v1.PersistentVolumeClaim, srcFormat, dstFormat string
 						SeccompProfile: &v1.SeccompProfile{
 							Type: v1.SeccompProfileTypeRuntimeDefault,
 						},
-						FSGroup: ptr.To(int64(107)),
+						RunAsNonRoot: ptr.To(true),
```

Comment:
I see that for the vddk-validator we get the following error when creating a plan that migrates to the default namespace:
```
Error: container has runAsNonRoot and image will run as root (pod: "vddk-validator-migplan-glhyj7m79-nk6zp_default(f028dd10-3c86-408f-a3a2-94ceb086a7f1)", container: validator)
```
and I see that this code changes the security context here to be similar to that of the vddk validator pod - do we need to also set a user?

Reply:
```
...
spec.securityContext.fsGroup: Invalid value: []int64{107}: 107 is not an allowed group, provider restricted-v2: .containers[0].runAsUser: Invalid value: 107: must be in the ranges: [1000780000, 1000789999],
...
```

Reply:
posted a fix for that issue - #771, we can do it similarly here

Reply:
what's different?

Reply:
ah seems that I've missed the setting of `RunAsUser` below, so just to replace 107 with more expressive const

Reply:
uh this doesn't actually work with non-default namespaces

Reply:
interesting, yeah - there's a problem with restricted namespace
we had this issue with the ova server, but we don't have this issue with the v2v pod...